hexrd/ui/color_map_editor.py

```diff
@@ -54,11 +55,26 @@ def data(self, v):
             self.update_bc_editor()
 
     def load_cmaps(self):
+        if not (cmaps := HexrdConfig().limited_cmaps_list):
+            cmaps = sorted(i[:-2] for i in dir(cm) if i.endswith('_r'))
+
+        with block_signals(self.ui.color_map):
+            old_selection = self.ui.color_map.currentText()
+            self.ui.color_map.clear()
+            self.ui.color_map.addItems(cmaps)
+
```

Comment:
The loaded list of cmaps and default setting were out of sync initially because I had incorrectly set the default value for the setting (fixed in [558c57e](https://github.com/HEXRD/hexrdgui/pull/1410/commits/558c57e9b25b91092a4ad96d10e99fcafd494a22)). I agree that it still makes sense to combine the two functions though - cleaned up!